chain/consensus/mir/manager.go

```diff
@@ -143,6 +153,19 @@ func NewManager(ctx context.Context, addr address.Address, api v1api.FullNode) (
 		return nil, fmt.Errorf("failed to create crypto manager: %w", err)
 	}
 
+	// Instantiate an interceptor.
+
+	var interceptor *eventlog.Recorder
+
+	interceptorOutput := os.Getenv(InterceptorOutputEnv)
```

Comment:
What is the interceptor? Should this only be started in "debugging mode" to avoid unnecessary overheads?

Reply:
Nevermind, I see below that this is the case already :)

Reply:
Yes, I also added that to the PR description.
I was trying to get an interceptor from the context but it is changed when Subnet is started, so I decided to use environment variables